### PR TITLE
Truncate laterpay_post_views table periodically.

### PR DESCRIPTION
**Pull request by AliaksandrVahura-ScienceSoft**
Truncate added. Fixed issue with plugin installation. Fixed issue with js scripts enqueue even if action not called.

### CODE REVIEW

File: laterpay/application/Core/Bootstrap.php [hunk 1]
```diff
@@ -423,6 +426,7 @@ public function activate() {
 
         // register the 'refresh dashboard' cron job
         wp_schedule_event( time(), 'hourly', 'laterpay_refresh_dashboard_data' );
+        wp_schedule_event( time(), 'daily', 'laterpay_delete_old_post_views' );
```
Comment (drdla) [inline laterpay/application/Core/Bootstrap.php]: It's not helpful to put this under the 'refresh dashboard' cron job with the same comment.
The comment should probably be something like "// register cron jobs".

File: laterpay/application/Model/Post/View.php [hunk 2]
```diff
@@ -299,4 +299,22 @@ public function get_sparkline( $post_id, $start_timestamp, $interval = 'week', $
...
+     * @return bool  $success
+     */
+    public function delete_old_data() {
+        global $wpdb;
+
+        $sql = "
+            DELETE FROM
+                {$this->table}
+            WHERE
+                date < DATE_SUB( CURDATE(), INTERVAL 3 MONTH );";
```
Comment (drdla) [inline laterpay/application/Model/Post/View.php]: Please don't hardcode the interval into the SQL.

Comment (AliaksandrVahura-ScienceSoft) [inline laterpay/application/Model/Post/View.php]: Ok. I will fix your notices.

**Comment by drdla** [comment]
@mturalenka good to merge now?

**Comment by mturalenka** [comment]
@drdla so we will remove old records older then 3 month on daily basis, is that ok?

**Comment by drdla** [comment]
Yes, that is the intended behaviour of that issue.

File: laterpay/application/Controller/Frontend/Invoice.php [hunk 3]
```diff
@@ -20,6 +20,9 @@ class LaterPay_Controller_Frontend_Invoice extends LaterPay_Controller_Base
      */
     public function the_invoice_indicator() {
         echo laterpay_sanitized( $this->get_text_view( 'frontend/partials/widget/invoice-indicator' ) );
+
+        wp_enqueue_script( 'laterpay-yui' );
+        wp_enqueue_script( 'laterpay-invoice-indicator' );
```
Comment (mturalenka) [inline laterpay/application/Controller/Frontend/Invoice.php]: @AliaksandrVahura-ScienceSoft this is not related to the ticket, its another issue we figured yesterday ;)
